libraries/CMakeLists.txt

```diff
@@ -0,0 +1 @@
+add_subdirectory(ctrlLibRT)
```

Comment:
The purpose of this library was to enable the use of low pass filters on the encoder measurements.
I think these filters were ported to icub-main which is a dependency for this repository.
This must be kept in mind to refactor out this library.

Reply:
>  I think these filters were ported to icub-main which is a dependency for this repository.

Unfortunately, these filters were forked out of icub-main to remove the parts of the interface that prevented an efficient implementation (mainly the use of `yarp::sig::Vector`) so they cannot be directly replaced by their icub-main corresponding class. However, this is again something that should be tracked in an issue.

Reply:
I didn't know that. Thank you for the information.